api/controllers/apps_test.go

```diff
@@ -47,7 +49,10 @@ func TestAppShow(t *testing.T) {
 }
 
 func TestAppShowWithAppNotFound(t *testing.T) {
-	aws := test.StubAws(test.DescribeStackNotFound("bar"))
+	aws := test.StubAws(
+		test.DescribeStackNotFound("bar"),
```

Comment:
I wonder if this is the right order... After this lands, the majority of apps will have the name prefix, so perhaps its more correct to look the `convox-test-bar` stack up first, and do the second lookup as a fallback for legacy apps.